test/test_pipeline_baseline.py

```diff
@@ -97,17 +109,114 @@ def _test_baseline(task_name, test_name, language, output_dir, extra_kwargs,
     check_calls_match(expected_subprocess_call, mock_call.mock_calls)
 
 
+@mock.patch('pipeline.utils.task_utils.runGradleTask')
+@mock.patch('subprocess.call')
+@mock.patch('subprocess.check_call')
+@mock.patch('os.chdir')
+def _test_cli_baseline(pipeline_name, config, pipeline_kwargs, baseline,
```

Comment:
It is extremely similar, so probably yes. The purpose is to test that all of the pipelines work correctly when they are run through the [ArtmanPipeline](https://github.com/googleapis/artman/pull/44/files#diff-d79f6027b58cc3550400394dcfebf88aR22) object rather than just run directly, as they are in the other tests.

Maybe this could replace the other tests?